content/en/docs/tasks/debug/debug-cluster/kubectl-node-debug.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Debugging Kubernetes nodes with kubectl
+content_type: task
+---
+
+<!-- overview -->
+This page shows how to debug a [node](/docs/concepts/architecture/nodes/) running on the Kubernetes cluster using `kubectl debug` command.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Debugging Node using kubectl debug node
+
+Use the `kubectl debug node` command to deploy a Pod to a Node that you want to troubleshoot.
+This command is helpful in scenarios where you can't access your Node by using an SSH connection.
+When the Pod is created, the Pod opens a interactive shell on the Node.
+To create an interactive shell on a Node using `kubectl debug`, run:
+
+```shell
+kubectl debug node/mynode -it --image=ubuntu
+```
+
+```shell
+Creating debugging pod node-debugger-mynode-pdx84 with container debugger on node mynode.
+If you don't see a command prompt, try pressing enter.
+root@ek8s:/#
+```
+
+Run debug commands to help you gather information and troubleshoot issues. Commands 
+that you might use to debug, such as `ip`, `ifconfig`, `nc`, `ping`, and `ps`. You can also
+install other tools, such as `mtr`, `tcpdump`, and `curl`, from the respective package manager.
+
+Get the following logs from the shell session to look for issues on the Node. Note that the root filesystem
+of the Node will be mounted at `/host`
```

Comment:
```suggestion
The root filesystem of the Node (as seen by the kubelet) is available into the debugging Pod, and is
mounted as `/host`. For a typical Linux node, you can look at the following paths to find relevant logs:
```

Reply:
How about

```suggestion
The debugging Pod can access the root filesystem of the Node, mounted at `/host`.
```

not sure if we need to mention "as seen by the kubelet" - if we do need to maybe we should make it clearer what is being seen by the kubelet

Reply:
If you run your kubelet inside a filesystem namespace, the debug pod sees the root _for that namespace_ not the node's root filesystem. That's what I want to emphasize.

Reply:
> Hmm so something like "The debugging Pod can access the root filesystem of the Node, mounted at `/host`." If you run your kubelet in a filesystem namespace, the debugging Pod sees the root for that namespace, not for the entire node. For a typical&hellip;"

That'd work!